.travis.yml

```diff
@@ -45,6 +46,12 @@ matrix:
     - env: PLATFORM=ios-10.0
       os: osx
       osx_image: xcode9
+    - env: PLATFORM=ios-11.3
+      os: osx
+      osx_image: $XCODE_VERSION
```

Comment:
that does not work :(

Reply:
Yep, `osx_image` doesn't seem to like ENV vars. Should be fixed now.